packages/jest-phabricator/src/index.js

```diff
@@ -53,9 +53,27 @@ module.exports = function(results: AggregatedResult): PhabricatorReport {
   const formatter = (coverage, reporter) => coverageMap;
   const report = formatTestResults(results, formatter);
 
-  return Object.assign((Object.create(null): any), results, {
+  const returnPhabricatorReport: PhabricatorReport = {
```

Comment:
What would be the syntax for that?  I'm about 99% new to Javascript (and Flow) -- this is a FB bootcamp task for me.  Happy to fix it up if there's a better way of addressing the "any" type in the original (this is the weak type that causes the error).  The '&' on the PhabricatorReport type is what led me to creating a new variable and adding in all of the elements specified in the types.  It's the only way I could figure out to get the types correct on what gets returned.  Side question: when I run "yarn" around 30 files in the project auto update for me.  The error from babel-jest is fixed, but I didn't include the 30 files in my second commit and it looks like some checks fail.  Should I add all updated files to the commit?  Thanks!